samples/javascript/KafkaTriggerRetry/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = async function (context, event) {
+    // context.log.info(event)
```

Comment:
Is this comment required?

Reply:
Captured in -- https://github.com/Azure/azure-functions-kafka-extension/pull/376